tests/cover/test_line.py

```diff
@@ -0,0 +1,9 @@
+from src.cover.cover import Line, Cover
```

Comment:
```suggestion
from src.cover.cover import Line
```
`Cover` is not used in this test case.